src/utils/barcode.ts

```diff
@@ -50,24 +47,27 @@ export function toBarcode(
     bcid,
     text
   };
-  return sequenceS(taskEither)({
-    png: toBufferPng(options),
-    svg: fromEither(toBufferSvg(options))
-  }).foldTaskEither(
-    errorOrString =>
-      fromEither(
-        left(
-          typeof errorOrString === "string"
-            ? new Error(errorOrString)
-            : errorOrString
+  return pipe(
+    AP.sequenceS(TE.ApplicativePar)({
```

Comment:
To be sure, did you check that `sequenceS(taskEither)` was a parallel execution?

Reply:
check done by the @BurnedMarshal and @gquadrati